projects/packages/forms/src/blocks/contact-form/components/jetpack-email-connection-settings.js

```diff
@@ -87,44 +87,51 @@ const JetpackEmailConnectionSettings = ( {
 
 	return (
 		<>
-			<InspectorHint>
-				{ __( 'Get incoming form responses sent to your email inbox:', 'jetpack-forms' ) }
-			</InspectorHint>
-			<TextControl
-				aria-describedby={ `contact-form-${ instanceId }-email-${
-					hasEmailErrors() ? 'error' : 'help'
-				}` }
-				label={ __( 'Email address to send to', 'jetpack-forms' ) }
-				placeholder={ __( 'name@example.com', 'jetpack-forms' ) }
-				onKeyDown={ e => {
-					if ( event.key === 'Enter' ) {
-						e.preventDefault();
-						e.stopPropagation();
-					}
+			<ToggleControl
+				checked={ !! sendEmailOnSubmission }
+				label={ __( 'Enable email notifications', 'jetpack-forms' ) }
+				onChange={ newValue => {
+					setAttributes( { sendEmailOnSubmission: newValue } );
 				} }
-				value={ emailAddress }
-				onBlur={ onBlurEmailField }
-				onChange={ onChangeEmailField }
-				help={ __(
-					'You can enter multiple email addresses separated by commas.',
-					'jetpack-forms'
-				) }
-				__nextHasNoMarginBottom={ true }
-				__next40pxDefaultSize={ true }
-			/>
-
-			<HelpMessage isError id={ `contact-form-${ instanceId }-email-error` }>
-				{ getEmailErrors() }
-			</HelpMessage>
-
-			<TextControl
-				label={ __( 'Email subject line', 'jetpack-forms' ) }
-				value={ emailSubject }
-				placeholder={ __( 'Enter a subject', 'jetpack-forms' ) }
-				onChange={ newSubject => setAttributes( { subject: newSubject } ) }
-				__nextHasNoMarginBottom={ true }
-				__next40pxDefaultSize={ true }
+				help={ __( 'Get incoming form responses sent to your email inbox.', 'jetpack-forms' ) }
 			/>
+			{ sendEmailOnSubmission && (
```

Comment:
There are no changes below. It's just indentation from the conditional rendering.